### ISSUE TITLE
About Training 

### ISSUE THREAD

**Issue by pinguo-huxiaohe** [zhanxinrui/HDN#7]
I tried two different training methods.
one is training without pertained model, but loss is like this and do not decrease
--------------------------------------------------------------------------------------------------------------------------------
epoch: 11 481/9375 lr: 0.00241 cls_loss_sup: 0.089103 loc_loss_sup: 0.042983 cls_loss_lp_sup: 0.000710 loc_loss_lp_sup: 0.006392 sim_cent_loss: 0.139188 sim_loss: 0.010180 cor_loss: 25.370169 cor_pos_loss: 25.370132 cor_neg_loss: 0.000036 homo_unsup_loss: 0.017426 total_loss: 15.785341 
epoch: 11 511/9375 lr: 0.00255 cls_loss_sup: 0.083010 loc_loss_sup: 0.036742 cls_loss_lp_sup: 0.000312 loc_loss_lp_sup: 0.005055 sim_cent_loss: 0.125119 sim_loss: 0.009541 cor_loss: 29.103842 cor_pos_loss: 29.103813 cor_neg_loss: 0.000029 homo_unsup_loss: 0.018057 total_loss: 14.545660 
epoch: 11 541/9375 lr: 0.00271 cls_loss_sup: 0.110210 loc_loss_sup: 0.056279 cls_loss_lp_sup: 0.000471 loc_loss_lp_sup: 0.010177 sim_cent_loss: 0.177137 sim_loss: 0.012793 cor_loss: 24.089970 cor_pos_loss: 24.089928 cor_neg_loss: 0.000042 homo_unsup_loss: 0.013832 total_loss: 19.386679 
epoch: 11 571/9375 lr: 0.00285 cls_loss_sup: 0.097187 loc_loss_sup: 0.030674 cls_loss_lp_sup: 0.006395 loc_loss_lp_sup: 0.005135 sim_cent_loss: 0.139392 sim_loss: 0.008259 cor_loss: 22.408772 cor_pos_loss: 22.408726 cor_neg_loss: 0.000046 homo_unsup_loss: 0.011788 total_loss: 16.211657 
epoch: 11 601/9375 lr: 0.00300 cls_loss_sup: 0.094609 loc_loss_sup: 0.069493 cls_loss_lp_sup: 0.001097 loc_loss_lp_sup: 0.006270 sim_cent_loss: 0.171469 sim_loss: 0.007646 cor_loss: 28.108177 cor_pos_loss: 28.108162 cor_neg_loss: 0.000015 homo_unsup_loss: 0.011420 total_loss: 19.361439 
epoch: 11 631/9375 lr: 0.00316 cls_loss_sup: 0.089650 loc_loss_sup: 0.033506 cls_loss_lp_sup: 0.004167 loc_loss_lp_sup: 0.005310 sim_cent_loss: 0.132633 sim_loss: 0.008574 cor_loss: 25.780205 cor_pos_loss: 25.780182 cor_neg_loss: 0.000022 homo_unsup_loss: 0.019004 total_loss: 15.041709 
epoch: 11 661/9375 lr: 0.00331 cls_loss_sup: 0.097170 loc_loss_sup: 0.057477 cls_loss_lp_sup: 0.001001 loc_loss_lp_sup: 0.005108 sim_cent_loss: 0.160755 sim_loss: 0.005386 cor_loss: 28.488239 cor_pos_loss: 28.488205 cor_neg_loss: 0.000035 homo_unsup_loss: 0.011990 total_loss: 18.341190 
--------------------------------------------------------------------------------------------------------------------------------
and I use the model to inference and got the bad results.
can you tell me the total_loss when you train the model?

another one is train model based on the pretrained model you provided and I also get the bad result.

Looking forward to your reply,please.

**Comment by pinguo-huxiaohe** [comment]
training datasets:GOT10K_E2E,GOT10K_E2E_UNSUP,COCO14
I have not change any configs of the training code.
